actions/is_valid_ip_port.py

```diff
@@ -30,8 +30,10 @@ def run(self, port):
         """
 
         if port < 0:
-            raise ValueError("Invalid port: {} is less than 0.".format(port))
+            raise ValueError("Invalid port: {} is less than 0.".format(
+                port))
         elif port > 65535:
-            raise ValueError("Invalid port: {} is greater than 65535.".format(port))
+            raise ValueError("Invalid port: {} is greater than 65535.".format(
+                port))
```

Comment:
Heh, just a heads up - we use 100 characters limit for the line length in all StackStorm Python code bases.